crates/ruff/src/rules/pycodestyle/snapshots/ruff__rules__pycodestyle__tests__tab_size_1.snap

```diff
@@ -0,0 +1,56 @@
+---
+source: crates/ruff/src/rules/pycodestyle/mod.rs
+---
+E501_2.py:1:81: E501 Line too long (89 > 88 characters)
+  |
+1 | a = """ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A6"""
+  |                                                                                         ^ E501
+2 | a   =   """ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A6"""
+  |
+
+E501_2.py:2:81: E501 Line too long (89 > 88 characters)
+  |
+2 | a = """ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A6"""
+3 | a   =   """ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A67ß9💣2ℝ4A6"""
+  |                                                                                             ^ E501
```

Comment:
Is the caret at the right position? It seems to point past the end?

Reply:
Github seems to render 💣 with only 1.5 width instead of 2 in VSCode and vim.
```text
💣💣
123
```